exporter/opentelemetry-exporter-otlp-proto-http/pyproject.toml

```diff
@@ -42,6 +42,12 @@ test = [
 [project.entry-points.opentelemetry_traces_exporter]
 otlp_proto_http = "opentelemetry.exporter.otlp.proto.http.trace_exporter:OTLPSpanExporter"
 
+[project.entry-points.opentelemetry_metrics_exporter]
+otlp_proto_http = "opentelemetry.exporter.otlp.proto.http.metric_exporter:OTLPMetricExporter"
+
+[project.entry-points.opentelemetry_logs_exporter]
+otlp_proto_http = "opentelemetry.exporter.otlp.proto.http._log_exporter:OTLPLogExporter"
```

Comment:
Do we want to expose the LogExporter before it is stable?

Reply:
I think this is fine. We do this for gRPC, and the auto instrumentation is not stable yet.